node/node.go

```diff
@@ -180,7 +190,11 @@ func (n *Node) Start(ctx context.Context) error {
 			return fmt.Errorf("failed to register the operator: %w", err)
 		}
 	} else {
-		n.Logger.Info("The node has successfully started. Note it's not opt-in to EigenDA yet (it's not receiving or validating data in EigenDA). To register, please follow the EigenDA operator guide section in docs.eigenlayer.xyz")
+		eigenDAUrl := "unsupported chain"
+		if n.ChainID.String() == goerliChainID {
+			eigenDAUrl = "https://goerli.eigenlayer.xyz/avs/eigenda"
+		}
+		n.Logger.Infof("The node has successfully started. Note: if it's not opted in on %s, then please follow the EigenDA operator guide section in docs.eigenlayer.xyz to register", eigenDAUrl)
```

Comment:
when it's unsupported, what's the best way to write it? I guess this probably not that useful for local testing.

Reply:
I think it's find to leave it - since operators will most likely run on proper chains